pkg/api/component_report_test.go

```diff
@@ -220,17 +230,11 @@ func TestGenerateComponentReport(t *testing.T) {
 		Arch:     "amd64",
 		Network:  "ovn",
 	}
-	columnAWSAMD64OVNVariantStandard := apitype.ComponentReportColumnIdentification{
+	columnAWSAMD64OVNVariantFipsStandard := apitype.ComponentReportColumnIdentification{
 		Platform: "aws",
 		Arch:     "amd64",
 		Network:  "ovn",
-		Variant:  "standard",
-	}
-	columnAWSAMD64OVNVariantFips := apitype.ComponentReportColumnIdentification{
-		Platform: "aws",
-		Arch:     "amd64",
-		Network:  "ovn",
-		Variant:  "fips",
+		Variant:  "standard, fips",
```

Comment:
It looks like this should not have a space after the comma if it's intended to match the db flat_variants col.